src/09-word-problems.js

```diff
@@ -19,14 +22,43 @@
  *  applyDiscount(1000, 9, true);
  *  //> 700
  */
-function applyDiscount(priceInCents, age, hasMembership) {}
+function applyDiscount(priceInCents, age, hasMembership) {
+  // 10 or younger = 10% off
```

Comment:
I love seeing this!  Great work pseudo-coding your plans before writing a function moving forward lets remove them before pushing though.